### DISCUSSION TITLE
[Doc] Add a friend projects section

### DISCUSSION THREAD

**Discussion by JeremyBP** [reactiveui/refit#1832] [Ideas]
Hi,

It would be useful to add a doc section talking about some of Refit's "friend" projects.
I mean libraries based on Refit or working with it.

Something like:

## Friend projects

- Generating code:
    - [REST API Client Code Generator](https://github.com/christianhelle/apiclientcodegen): A collection of Visual Studio C# custom tool code generators for Swagger / OpenAPI specification files, including [Refitter](https://github.com/christianhelle/refitter).
    - [Refitter](https://github.com/christianhelle/refitter): Generate the Refit interface and contracts from Swagger / OpenAPI specifications, plus some optional registration helpers with or without [Apizr](https://apizr.net)
- Extending features:
    - [Apizr](https://apizr.net): Refit's client resilient manager (fault handling, connectivity checking, cache management, authentication handling, http tracing, priority management, data mapping and more...)
    - [Refit.Insane.PowerPack](https://github.com/thefex/Refit.Insane.PowerPack): Refit library extensions which provides attribute based cache and auto-retry features

Let me know if you want me to push a PR and if you know some other libs to be listed here.

**Comment by Shen37** [comment]
Hello can we have a discussion about this if yes kindly reach to me on my discord by "shen.li"
